compose-builder/README.md

```diff
@@ -1,28 +1,33 @@
 ## Edgex Docker Compose Builder
 
-This folder contains the `Compose Builder` which is made up of **source** compose and environment files and **makefile** for building the single file docker composes files for the configured `release`. The default release is `pre-release` and the generated compose files are placed in `releases/pre-release`. 
+This folder contains the `Compose Builder` which is made up of **source** compose, environment files and a **makefile** for building the single file docker composes files for the configured `release`. The default release is `pre-release` and the generated compose files are placed in in the top level of this repository. 
 
 > **Note to Developers**: 
-> *Once you have edited and tested your changes to these source files you **MUST** regenerate the composes using the `make build` command.*
+> *Once you have edited and tested your changes to these source files you **MUST** regenerate the standard `pre-release` compose files using the `make build` command.*
 
 ### Generate next release compose files
 
-Do the following to build compose files for next release such as `hanoi` 
+Do the following to generate the compose files for next release such as `ireland` 
 
-1. Update the `RELEASE`, `REPOSITORY`, `CORE_EDGEX_REPOSITORY`, `APP_SVC_REPOSITORY`, and `versions` contained in the `.env` file.
-2. Create the release folder, i.e `release/hanoi`
+1. Create the new `release` branch from this branch, i.e create the `ireland` branch
+2. Checkout a new working branch from the new `release` branch
+3. Update the `RELEASE`, `REPOSITORY`, `CORE_EDGEX_REPOSITORY`, `APP_SVC_REPOSITORY`, and `versions` contained in the `.env` file appropriately for the new release
+4. Run `make build` 
+5. Commit changes, open PR and merge PR
+6. Update documentation for the new release to refer to the new release branch.
+
+### Generate dot release compose files
+
+1. Checkout a new working branch from the target `release` branch for the dot release, i.e the `ireland` branch
+2. Update the and `versions` contained in the `.env` file appropriately for the dot release
 3. Run `make build` 
 4. Commit changes, open PR and merge PR
-5. TAG repo for new release, i.e. `1.3.0`
-   - This is need for generating dot releases and allows user to get back to the `release` source compose and .env files so they can generate custom compose files base on the `release`.  
```

Comment:
I think tagging still have some benefits and values like you can track back the official releases on a certain version of that branch instead of relying on the branches and commits solely.

Reply:
Ok, I'll add that back.